### PR TITLE
Delegated Admin + Hooks Update

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

**Comment by fyockm** [comment]
@khorne3 we are finally ready to deploy DAE 3.0! 👏  Do you think these docs updates could go out this week?

**Comment by khorne3** [comment]
@fyockm Sure thing! Let me rebase and we should be good to go. Do you know if there are any content changes that need to be made?

File: articles/extensions/delegated-admin/v3/manage-users.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  - how-to
+  - concept
+useCase: extensibility-extensions
+---
+
+# Delegated Administration: Manage Users
+
+In the Application exposed by the Delegated Administration extension, there are two views available: *Users* and *Logs*. On the *Users* view, you can see the display and modify users associated with your Auth0 account.
+
+By default, all users are displayed, but you can filter the displayed list by configuring a [filter hook](/extensions/delegated-admin/hooks#the-filter-hook).
```
Comment (fyockm) [inline articles/extensions/delegated-admin/v3/manage-users.md]: link should be `/extensions/delegated-admin/hooks/v3/filter`

File: articles/extensions/delegated-admin/v3/manage-users.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Available User Actions in the Delegated Administration Dashboard
+
+The table below lists the options you can perform on users, as well as information on whether the option is available via the [Management Dashboard](${manage_url}/#/) and/or the Delegated Administration extension. To limit the number of options someone with access to the Dashboard exposed by the Delegated Administration extension, configure an [access hook](#access-hook).
```
Comment (fyockm) [inline articles/extensions/delegated-admin/v3/manage-users.md]: link should be `extensions/delegated-admin/hooks/v3/access`

File: articles/extensions/delegated-admin/v3/hooks/index.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+## Available Hooks
+
+The following Hooks are available for use with your Delegated Administration extension:
+
+* [The Access Hook](/extensions/delegated-admin/v3/hooks/access)
+* [The Filter Hook](/extensions/delegated-admin/v3/hooks/filter)
+* [The Memberships Query Hook](/extensions/delegated-admin/v3/hooks/membership)
+* [The Settings Query Hook](/extensions/delegated-admin/v3/hooks/settings)
+* [The Write Hook](/extensions/delegated-admin/hooks/v3/write)
```
Comment (fyockm) [inline articles/extensions/delegated-admin/v3/hooks/index.md]: Write Hook link should be `/extensions/delegated-admin/v3/hooks/write`

File: articles/extensions/delegated-admin/v3/hooks/access.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  - users
+  - hooks
+contentType:
+  - how-to
+  - concept
+useCase: extensibility-extensions
+---
+# Delegated Administration Hooks: The Access Hook
+
+Because the [Filter Hook](/extensions/delegated-admin-hooks/filter) only applies filtering logic, you'll need a second layer of logic to determine if the current user (or the person acting as the administrator) is allowed to access a specific user.
```
Comment (fyockm) [inline articles/extensions/delegated-admin/v3/hooks/access.md]: Link -> `/extensions/delegated-admin/hooks/v3/filter`

**Review by fyockm — COMMENTED** [review]
Just a few incorrect link

**Review by fyockm — APPROVED** [review]
LGTM 🚀

**Comment by khorne3** [comment]
@fyockm When do you want me to merge?

**Comment by fyockm** [comment]
@khorne3 today if possible. I'm ready to roll out the DAE 3.0 release. Was just waiting on the docs. Don't think they need to be perfectly in sync, but close.

**Pull request merged**